Include/cpython/optimizer.h

```diff
@@ -89,7 +89,7 @@ static inline uint16_t uop_get_error_target(const _PyUOpInstruction *inst)
 
 typedef struct _exit_data {
     uint32_t target;
-    int16_t temperature;
+    _Py_BackoffCounter temperature;
```

Comment:
`temperature` is now a bit of a misnomer, since it counts down. Maybe it should be renamed to `counter` (same as in `CODEUNIT`)?

Reply:
I'm fine with either.

Reply:
I'll keep `temperature`, despite the misnomer -- it stands out and makes it easy to grep for this particular concept.